rust/stackablectl/src/cli/mod.rs

```diff
@@ -290,3 +291,27 @@ fn get_files(default_file: &str, env_key: &str) -> Result<Vec<PathOrUrl>, PathOr
 
     Ok(files)
 }
+
+#[derive(Clone, Debug, Default, ValueEnum)]
+pub enum ChartSourceTypeArg {
+    /// OCI registry
+    #[default]
+    OCI,
+
+    /// Nexus repositories: resolution (dev, test, stable) is based on the version and thus will be operator-specific
```

Comment:
suggestion: Drop the term "Nexus" and replace it with "index.yaml-based". The "old" Helm repositories are _not_ related to Nexus, but instead use a webserver which serves the `index.yaml` file listing available charts.

Reply:
d4e3b32